public/js/practise.js

```diff
@@ -41,6 +41,7 @@ function CompareNumberIfElse(number1, number2) {
 console.log(compare);
 
 //Ex 4 factorialul unui numar cu functie si parametru//
```

Comment:
In this exercise, try to find a solution without using factorial() function from Javascript.
You need to read about **for()** statement in Javascript, link down below.
[https://www.w3schools.com/js/js_loop_for.asp](url)

Think how you can use **for()** to loop thru a number.
For example if user put number in your function **factorial(3)** , then somehow you function must go from 1 to 3, like this 1 * 2 * 3 , and if user uses **factorial(100)** your function must take that number **100** and go from **1 * 2 * 3 * 4 * 5 * .. * 100** and return the result of adding each number.